src/components/NewCommentForm.tsx

```diff
@@ -45,24 +144,36 @@ export const NewCommentForm: React.FC = () => {
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': isEmailError })}
+            value={newComment.email}
+            onChange={(event) => {
```

Comment:
again, don't use anonymous function

Reply:
Not resolved. Just move all arrow functions to the variables.